src/Fleece.FSharpData/Fleece.FSharpData.fsproj

```diff
@@ -1,19 +1,23 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
-    <NoWarn>0686</NoWarn>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Description>JSON mapper for FSharp.Data</Description>
-    <DefineConstants>FSHARPDATA;$(DefineConstants)</DefineConstants>
-    <OtherFlags>--warnon:1182</OtherFlags>    
-  </PropertyGroup>
-  <ItemGroup>
-    <Compile Include="../Fleece/Fleece.fs" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Update="FSharp.Core" Version="4.6.2" />
-    <PackageReference Include="FSharpPlus" Version="1.1.1" />
-    <PackageReference Include="FSharp.Data" Version="3.0.0" />
-  </ItemGroup>
-</Project>
\ No newline at end of file
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+        <NoWarn>0686</NoWarn>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <Description>JSON mapper for FSharp.Data</Description>
+        <DefineConstants>FSHARPDATA;$(DefineConstants)</DefineConstants>
+        <OtherFlags>--warnon:1182</OtherFlags>
+    </PropertyGroup>
+    <ItemGroup>
+        <Compile Include="../Fleece/Fleece.fs" />
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Update="FSharp.Core" Version="4.6.2" />
+        <PackageReference Include="FSharpPlus" Version="1.1.1" />
+        <PackageReference Include="FSharp.Data" Version="3.0.0" />
+        <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0">
```

Comment:
I don't think you need to add this package since you already have a reference in Directory.Build.props

Reply:
Oh, great point :+1: